src/bin/cargo/cli.rs

```diff
@@ -491,9 +491,9 @@ pub fn cli() -> Command {
     #[allow(clippy::disallowed_methods)]
     let is_rustup = std::env::var_os("RUSTUP_HOME").is_some();
     let usage = if is_rustup {
-        "cargo [+toolchain] [OPTIONS] [COMMAND]"
+        "cargo [+toolchain] [OPTIONS] [COMMAND]\n       cargo [+toolchain] [OPTIONS] -Zscript <MANIFEST_RS> [ARGS]..."
```

Comment:
So people on stable will see this as well?

```
Usage: cargo [OPTIONS] [COMMAND]
       cargo [OPTIONS] -Zscript <MANIFEST_RS> [ARGS]...
```

nit: should we make this string a bit better to read? There are spaces I cannot count 😆.

Reply:
I went ahead and added this to the usage, despite being unstable, because we document other unstable CLI behavior (like unstable-only args) in `--help`.

If you have a thought on how better to handle the spaces, I'd be open.  Longer term, I want clap to support being given multiple usages and letting it figure out the formatting.